service/main.go

```diff
@@ -51,12 +52,11 @@ func defaultAppName() string {
 }
 
 func run(f flags) {
-	ulog := logger{}
-
 	if f.version {
-		ulog.Infof("%s\n", updater.Version)
+		fmt.Printf("%s\n", updater.Version)
```

Comment:
This is unrelated but wanted to fix how we output the version. This allows 
```sh
updater_version=`updater -version`
```
to work in scripts